rhalphalib/parameter.py

```diff
@@ -205,8 +205,7 @@ def __init__(self, name, formula, *dependents):
 
     @property
     def value(self):
-        # TODO: value from rendering formula and eval() or numexpr or TFormula or ...
-        raise NotImplementedError
+        return eval(self.formula().replace("_smoothstep", "").format(**{p.name: p.value for p in self.getDependents(deep=True)}))
```

Comment:
Would implementing https://github.com/nsmith-/rhalphalib/blob/d053fadb87093ed4bc7d1557e51c19e311ebaadd/rhalphalib/parameter.py#L274-L276 avoid the find-replace for `_smoothstep`?

Reply:
The issue was iirc that `getDependents` wasn't returning params with `_smoothstep` in the names

Reply:
I can't recall why the `SmoothStep` is its own class, but I would worry that without implementing its `value()` method we aren't actually interpolating correctly in the eval.

Reply:
Ok, hold on on merging this. I'll take another look